scripts/core/auth/auth-service.js

```diff
@@ -19,4 +19,25 @@ angular.module('superdesk.core.auth.auth', []).service('auth', ['$q', 'api', 'se
                         return session.identity;
                     }));
         };
+
+
+    /**
+     * Login using given credentials
```

Comment:
You don't _have_ to fix this, but it would be nice if you could make sure this documentation actually shows up when running `grunt docs`.

I know most of this file is the wrong format, but if we take baby steps we can slowly convert it, so any help is much appreciated. For this to show up, the hierarchy would have to be in place, so you would have to document the top service too. You could take a look at [SuggestService](https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/authoring/suggest/SuggestService.js) for guidance.

Feel free to ignore this comment if you don't find the time to do this.